charts/perses/values.yaml

```diff
@@ -142,17 +152,17 @@ config:
       extension: json
     # -- SQL Config
     # sql:
-    ## user: "user"
-    ## password: "password"
-    ## net: "tcp"
-    ## addr: "localhost:3306"
-    ## db_name: "perses"
-    ## allow_native_passwords: true
+    #   user: "user"
+    #   password: "password"
+    #   net: "tcp"
+    #   addr: "localhost:3306"
+    #   db_name: "perses"
+    #   allow_native_passwords: true
 
   # -- provisioning config
   provisioning:
     folders:
-    - /etc/perses/provisioning
+      - /etc/perses/provisioning
     interval: 10s
```

Comment:
that's really a low level interval. By default it's much more. Like 1 hour or something like that

Reply:
having a such low interval if there is a lot of resource to provision can be harmful for the database and the system

Reply:
I will create a follow-up PR and I will increase the default interval :)